### DISCUSSION TITLE
Make the logout and stop impersonating confirm windows optional

### DISCUSSION THREAD

**Discussion by seabasss** [laravel/nova-issues#4176] [Ideas & Feature Requests]
Would love settings to turn off the confirm windows when you log out and when you stop impersonating.

Thanks!

**Comment by ziming** [comment]
how do you exit impersonation in nova 4?

**Reply by seabasss** [reply]
Click on the name at the top right and click stop impersonating.

**Reply by ziming** [reply]
can show a screenshot, because I did not see such a thing when I impersonate in nova 4

**Reply by seabasss** [reply]
<img width="314" alt="Screen Shot 2022-05-05 at 7 32 14 PM" src="https://user-images.githubusercontent.com/5251896/167057304-9aa2eabb-a0d9-4d93-a5cc-2e5f040c57d9.png">

**Reply by schonhoff** [reply]
If the impersonated user has no Nova Access you need to implement something like this somewhere in your frontend:

```php
@if(app(\Laravel\Nova\Contracts\ImpersonatesUsers::class)->impersonating(request()))
  <a href="{{ route('impersonation.stop') }}" title="Stop impersonating">Stop imperonating</a>
@endif
```

**Reply by ziming** [reply]
> If the impersonated user has no Nova Access you need to implement something like this somewhere in your frontend:
> 
> ```php
> @if(app(\Laravel\Nova\Contracts\ImpersonatesUsers::class)->impersonating(request()))
>   <a href="{{ route('impersonation.stop') }}" title="Stop impersonating">Stop imperonating</a>
> @endif
> ```

thanks so much for this! Would be nice if it can be cleaner but for now I shall use this.
